app/scripts/controllers/landing.js

```diff
@@ -44,10 +45,26 @@ angular.module('confRegistrationWebApp')
         };
         $scope.searchComplete = val;
         $scope.searchResults = response.data;
+
+        $scope.filterUpdate();
       });
     }, 500, {leading: false});
 
-    $scope.dateFilter = function(event){
+    $scope.filterSearchResults = function(){
+      return _.filter($scope.searchResults, event => {
+        return dateFilter(event) && locationFilter(event);
+      });
+    };
+
+    $scope.filterUpdate = function(){
+      analytics.digitalData.searchTerm = $scope.searchVal;
+      analytics.digitalData.searchLocationFilter = $scope.eventFilters.locationName;
+      analytics.digitalData.searchDateFilter = $scope.eventFilters.date;
```

Comment:
@OzzieOrca I don't think we should push to prod until we get page titles. I'll check out the search direct rule call. I didn't know one was firing. I can probably just based the search stuff on that direct call rule then.